thoth/storages/graph/postgres.py

```diff
@@ -406,6 +406,14 @@ def get_alembic_version_count_all(self) -> int:
             result = session.execute(query).fetchone()
             return result[0]
 
+    def get_last_solver_datetime(self) -> datetime:
+        """Get the datetime of the last solver run synced in the database."""
+        query = "SELECT datetime FROM solver WHERE datetime IN (SELECT max(datetime) FROM solver)"
```

Comment:
Could this be rewritten to an SQLAlchemy query? It should be possible, something like (haven't tried it):

```python
from sqlalchemy.sql.expression import func

with self._session_scope() as session:
    result = session.query(func.max(Solver.datetime))
# ...
```

Reply:
Sure, thanks! :+1: